cypress/support/commands.ts

```diff
@@ -0,0 +1,37 @@
+/// <reference types="cypress" />
```

Comment:
we should keep this?

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/pull/8/commits/13af02d9fb3fdcdcf75ddd7887f7af7b9c690fee